docs/package.json

```diff
@@ -41,7 +41,7 @@
     "react-draggable": "^4.4.5",
     "react-native": "^0.71.4",
     "react-native-gesture-handler": "^2.10.0",
-    "react-native-reanimated": "3.5.0-nightly-20230829-dd8e1d530",
+    "react-native-reanimated": "3.6.0-nightly-20230928-2216f91c4",
```

Comment:
Change to `yarn.lock` is missing in this PR. Please commit changes to `yarn.lock` too